docs/source/api/asns.rst

```diff
@@ -32,11 +32,20 @@ Request Structure
 -----------------
 .. table:: Request Query Parameters
 
-	+----------------+----------+-----------------------------------------------------------------------------------------------------+
-	| Parameter      | Required |                                 Description                                                         |
-	+================+==========+=====================================================================================================+
-	|   cachegroup   | no       | An integral, unique identifier for a Cache Group - only ANSs for this Cache Group will be returned. |
-	+----------------+----------+-----------------------------------------------------------------------------------------------------+
+	+------------+----------+-----------------------------------------------------------------------------------------------------+
+	| Parameter  | Required | Description                                                                                         |
+	+============+==========+=====================================================================================================+
+	| cachegroup | no       | An integral, unique identifier for a Cache Group - only ANSs for this Cache Group will be returned. |
+	+------------+----------+-----------------------------------------------------------------------------------------------------+
+	| orderby    | no       | Choose the ordering of the results - must be the name of one of the fields of the objects in the    |
+	|            |          | ``response`` array                                                                                  |
+	+------------+----------+-----------------------------------------------------------------------------------------------------+
+	| sortOrder  | no       | Changes the order of sorting. Either ascending (default) or descending ("desc")                     |
```

Comment:
So is the only allowed value for this parameter `desc`, or can I use `asc` for the default behavior? Even if that is probably just a waste of time to type.

Reply:
If the parameter isn't `desc`, it's just ignored. So you could use `asc` if you wanted, but you could also use `asdfghjkl` with the same result